src/clj/game/cards/resources.clj

```diff
@@ -993,7 +993,9 @@
                                                                   (count from) from) card nil)
                      (do (clear-wait-prompt state :corp)
                          (effect-completed state side eid card)))))
-    :trash-effect {:effect (effect (mill :runner 3))}}
+    :trash-effect {:msg (msg "trash " (join ", " (map :title (take 3 (:deck runner))))
+                             " from their Stack as Rolodex was trashed ")
```

Comment:
Hmm i think it might need to move away from using the msg macro to have it make sense.  I will fix https://github.com/mtgred/netrunner/issues/2352 while I am at it.  Needs the same love anyway.